src/events.ts

```diff
@@ -44,6 +42,35 @@ export enum JwtSecretUpdateError {
   APIGatewayUpdateFailed,
 }
 
+export enum DatabaseUpgradeStatus {
+  Upgrading,
+  Upgraded,
+  Failed,
+}
+
+export enum DatabaseUpgradeProgress {
+  Started,
+  LaunchedUpgradedInstance,
```

Comment:
Based on experience with the JWT progress - should we consider making these string values?

i.e. 
```
Started = '1_started',
LaunchedUpgradedInstance = '2_launched_upgraded_instance',
```
?

Downside is more space consumption in events. Upside is the events are more consumable (vs for JWT rolls you just get e.g. 'progress: 2', 'progress: 4')

Reply:
Agreed - that makes debugging a lot less painful. Updated